challenge_one/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "nethermind-bot-deploy-and-update-monitor",
+  "version": "0.0.1",
+  "description": "Forta Agent Typescript starter project",
```

Comment:
This can be updated.

Also, you can add a `"repository": "url of the repo after merge"`  field, so the repo is automatically parsed and display on Forta UI after the bot's deployment.